pygitguardian/client.py

```diff
@@ -57,6 +73,27 @@ def is_ok(resp: Response) -> bool:
     )
 
 
+def create_tar(root_path: Path, filenames: List[str]) -> bytes:
```

Comment:
Can you rename this function to `_create_tar`? I prefer not making it part of pygitguardian public API.

Reply:
Done!